otx/mpa/modules/models/segmentors/otx_encoder_decoder.py

```diff
@@ -52,10 +52,16 @@ def single_stage_detector__extract_feat(ctx, self, img):
     def single_stage_detector__simple_test(ctx, self, img, img_metas, **kwargs):
         # without output activation
         #  seg_logit = self.encode_decode(img, img_metas)
-        #  feature_vector = FeatureVectorHook.func(self.feature_maps)
-        #  return seg_logit, feature_vector
+        # if ctx.cfg["dump_features"]:
```

Comment:
I don't know why this code was commented in the first place, but if it will be uncommented, I'd like these changes to be applied to this version as well. So I updated commented code as well

Reply:
I'd delete such comments. Barely we can find the original author to ask fixing that, because of refactoring/files moving operations. If someone ignores the dump_features flag, the tests will catch it.

Reply:
Okay, then I'll remove this comment, thank you